src/components/DocPage/DocPage.tsx

```diff
@@ -359,6 +370,43 @@ class DocPage extends React.Component<DocPageInnerProps, DocPageState> {
         );
     }
 
+    private renderNotification() {
+        const {notification, notificationCb} = this.props;
+        const {showNotification} = this.state;
+
+        if (!notification || !showNotification) {
+            return null;
+        }
+        const {title = '', content = '', type = ''} = notification;
+        const isNoteTypeCorrect = ['info', 'tip', 'warning', 'alert'].includes(type.toLowerCase());
+
+        return (
+            <div className={bNote('wrapper')}>
+                <div
+                    className={bNote(
+                        {},
+                        isNoteTypeCorrect ? `dc-accent-${type}` : bNote('template'),
+                    )}
+                >
+                    {title && <p className={bNote('title')}>{title}</p>}
+                    <Button
+                        view={'flat'}
+                        className={bNote('xmark')}
+                        onClick={() => {
+                            if (notificationCb) {
```

Comment:
it’s needed to be exported into useCallback

Reply:
this is a class component, we will do the refactoring into a functional one as a separate task